samples/wifi/scan/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(nrf_wifi_scan)
+
+set(WIFI_NRF_DRV_BASE ${WEST_TOPDIR}/nrf/drivers/wifi/nrf700x)
+
+if(CONFIG_WPA_SUPP)
+set(WPA_SUPP_BASE ${WEST_TOPDIR}/modules/lib/hostap)
+endif()
+
+target_compile_definitions_ifdef(CONFIG_WPA_SUPP_CRYPTO
+	app
+	PRIVATE
+	CONFIG_WEP
+)
```

Comment:
Remove, this is only for CONFIG_WPA_SUPP

Reply:
Addressed.